index.bs

```diff
@@ -2495,7 +2509,7 @@ error.
 :: `appid`
 
 : Client extension input
-:: A single [=UTF-8 encoded=] string specifying a FIDO |appId|.
+:: A single JSON string specifying a FIDO |appId|.
```

Comment:
To be nitpicky, since the extension input is a Javascript object, it's a [`{{DOMString}}`](https://heycam.github.io/webidl/#idl-DOMString) rather than a JSON string. JSON is a serialization of many Javascript objects, but it's not the type of things that get passed to web APIs.

No need to fix this before merging, just mentioning it for future patches.